src/main/kotlin/com/atlassian/performance/tools/report/api/judge/RelativeNonparametricPerformanceJudge.kt

```diff
@@ -60,9 +60,10 @@ class RelativeNonparametricPerformanceJudge(
             )
         val test = ShiftedDistributionRegressionTest(baseline, experiment, mwAlpha = significance, ksAlpha = 0.0)
         return if (test.isExperimentRegressed(toleranceRatio.toDouble())) {
-            val message = "Regression in [$label] is larger than allowed ${toleranceRatio.toPercentage()} tolerance at $significance significance level"
+            val confidenceLevelPercent = (1.0 - significance).toPercentage(decimalPlaces = 2)
+            val message = "There is $confidenceLevelPercent chance there is a regression. Possible regression is larger than allowed ${toleranceRatio.toPercentage(decimalPlaces = 2)} tolerance"
```

Comment:
Is that true? Do we have a source for this interpretation of Mann-Whitney alpha?

Reply:
@dagguh 
Based on
https://blog.minitab.com/en/adventures-in-statistics-2/understanding-hypothesis-tests-confidence-intervals-and-confidence-levels#:~:text=The%20significance%20level%20defines%20the,limits%20are%20to%20sample%20mean.

> The confidence level is equivalent to 1 – the alpha level. So, if your significance level is 0.05, the corresponding confidence level is 95%.

Reply:
Nice, then let's say "with confidence level" rather than "chance there is a regression" (implying a probability).

> For a specific confidence interval from one study, the interval either contains the population value or it does not—there’s no room for probabilities other than 0 or 1.